src/lib/jsonParseV2/jsonRequestTreat.cpp

```diff
@@ -74,10 +74,10 @@ std::string jsonRequestTreat
       return answer;
     }
 
-    if ((answer = parseDataP->ent.res.check(ciP, EntitiesRequest)) != "OK")
+    if ((answer = parseDataP->ent.res.check(ciP->apiVersion, EntitiesRequest)) != "OK")
     {
       OrionError oe(SccBadRequest, answer);
-      return oe.setStatusCodeAndSmartRender(ciP);
+      return oe.setStatusCodeAndSmartRender(ciP->apiVersion, &(ciP->httpStatusCode));
```

Comment:
() in &(ciP->httpStatusCode) are not necessary.

However, this is a question for the style-guide.

Personally I find it ugly (I am not used) but at the same time I understand it makes the thing easier to understand (for people that aren't aware of the operator precedence of C).

Let's add it to 'open questions' for the style guide.

NTC